datadog_checks_dev/datadog_checks/dev/tooling/e2e/docker.py

```diff
@@ -79,6 +79,9 @@ def exec_command(self, command, **kwargs):
         if kwargs.pop('interactive', False):
             cmd += ' -it'
 
+        if command.startswith('pip '):
+            command = command.replace('pip ', get_pip_exe(self.python_version), 1)
```

Comment:
how is this working? you cannot replace a string with a list

Reply:
`command` is a string, see line 86

Reply:
but get_pip_exe(self.python_version) returns an array

Reply:
Nice, thanks! https://github.com/DataDog/integrations-core/pull/4225